app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -89,6 +90,7 @@
     @BindView(R.id.card_view_nearby) public NearbyNotificationCardView nearbyNotificationCardView;
     @BindView(R.id.campaigns_view) CampaignView campaignView;
     @BindView(R.id.limited_connection_enabled_layout) LinearLayout limitedConnectionEnabledLayout;
+    @BindView(R.id.limited_connection_description_tv) TextView limitedConnectionDescriptionTv;
```

Comment:
Would you mind using the full words text_view and TextView instead of `tv`? Thanks!